### PR TITLE
[BVTCK-201] submit compatible value for validation

### PR DESCRIPTION
**Pull request by mbenson**

### CODE REVIEW

File: tests/src/main/java/org/hibernate/beanvalidation/tck/tests/validation/ValidateValueTest.java [hunk 1]
```diff
@@ -178,7 +179,7 @@ public void testValidIsNotHonoredValidateValue() {
 		customer.addOrder( order );
```
Comment (gunnarmorling) [inline tests/src/main/java/org/hibernate/beanvalidation/tck/tests/validation/ValidateValueTest.java]: Given this is testing `validateValue()`, the `customer` instance could be removed altogether.

Comment (mbenson) [reply tests/src/main/java/org/hibernate/beanvalidation/tck/tests/validation/ValidateValueTest.java]: Sure, I can make that change if you like.

**Comment by gsmet** [comment]
Squashed the commits and merged.

**Comment by gsmet** [comment]
Thanks Matt. FYI, our convention is to simply use the JIRA issue number in front of the commit message. We don't use square brackets.

**Pull request closed without merging**